.github/actions/patch-dependencies/action.yml

```diff
@@ -25,19 +27,19 @@ runs:
   steps:
     - name: check patches
       run: |
-        if [[ -f .github/patchs/opentelemetry-java.patch ]]; then
+        if [[ -f .github/patches/${{ inputs.branch }}/opentelemetry-java.patch ]]; then
           echo 'patch_otel_java=true' >> $GITHUB_ENV
         fi
-        if [[ -f .github/patchs/opentelemetry-java-instrumentation.patch ]]; then
+        if [[ -f .github/patches/${{ inputs.branch }}/opentelemetry-java-instrumentation.patch ]]; then
             echo 'patch_otel_java_instrumentation=true' >> $GITHUB_ENV
         fi
-        if [[ -f .github/patchs/opentelemetry-java-contrib.patch ]]; then
+        if [[ -f .github/patches/${{ inputs.branch }}/opentelemetry-java-contrib.patch ]]; then
             echo 'patch_otel_java_contrib=true' >> $GITHUB_ENV
         fi
       shell: bash
```

Comment:
nit (Wont block PR): I think this logic could be moved into the `patch.sh` script now that we use a `version` file to determine if patching is necessary. If a patch file exists it could set the output to the github env.